Assets/Scripts/ProgressManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public class ProgressManager: Singleton<ProgressManager>
+{
+    // Comments below is done in Mission Manager
+        // Keep track of all obtained missions
+        // Mission is obtained from an interactable
+        // Missions should also be an interactable which checks if mission is obtained (must talk with group)
```

Comment:
Oh, why might missions need to be an interactable?

Reply:
It was the whole thing we were talking about if triggering the mission needed to be an interactable or just something on enter and indicated by the UI. Left that comment there for more of a reminder

Reply:
Still not 100% convinced about it, cause the mission itself I imagine probably should not be an interactable, but an interactable (such as the end of an NPC interaction) will start the mission.

Reply:
Sorry, it was a poorly made comment but to clarify, I was referring to the entrance of the mission that should be the interactable and not the mission itself.